pkg/controllers/provisioning/v1alpha1/reallocation/terminate.go

```diff
@@ -147,7 +147,7 @@ func (t *Terminator) getNodes(ctx context.Context, provisioner *v1alpha1.Provisi
 	if err := t.kubeClient.List(ctx, nodes, client.MatchingLabels(functional.UnionStringMaps(map[string]string{
 		v1alpha1.ProvisionerNameLabelKey:      provisioner.Name,
 		v1alpha1.ProvisionerNamespaceLabelKey: provisioner.Namespace,
-	}, provisioner.Spec.Labels, additionalLabels))); err != nil {
+	}, additionalLabels))); err != nil {
```

Comment:
It does seem like we have the exact same code here as in *utilization.go* - could we eliminate the duplication?

Reply:
It is the same code. I have a task to de-duplicate these functions. 

Brandon, for now, can you also change the code in `utilization.go`? I'll de-dup in a follow-up.